samples/sensor/stm32_vbat_sensor/src/main.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+#include <sys/printk.h>
```

Comment:
```suggestion
#include <zephyr/zephyr.h>
#include <zephyr/device.h>
#include <zephyr/drivers/sensor.h>
#include <zephyr/sys/printk.h>
```

Cf https://github.com/zephyrproject-rtos/zephyr/pull/45388